pkg/controller/kogitoapp/builder/build_config.go

```diff
@@ -4,21 +4,29 @@ import (
 	v1alpha1 "github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 )
 
+// BuildType which build can we perform? Supported are s2i and service
+type BuildType string
+
 const (
-	kindImageStreamTag = "ImageStreamTag"
-	tagLatest          = "latest"
-	// ImageStreamTag default tag name for the ImageStreams
-	ImageStreamTag = "0.3.0"
-	// ImageStreamNamespace default namespace for the ImageStreams
-	ImageStreamNamespace = "openshift"
 	// BuildTypeS2I source to image build type will take a source code and transform it into an executable service
 	BuildTypeS2I BuildType = "s2i"
-	// BuildTypeService will create a image with a Kogito Service available
-	BuildTypeService BuildType = "service"
+	// BuildTypeRuntime will create a image with a Kogito Service available
+	BuildTypeRuntime BuildType = "runtime"
+	// BuildTypeRuntimeJvm will create a image with JRE installed to run the Kogito Service
+	BuildTypeRuntimeJvm BuildType = "runtime-jvm"
 	// LabelKeyBuildType is the label key to identify the build type
 	LabelKeyBuildType = "buildtype"
 )
 
+const (
+	kindImageStreamTag = "ImageStreamTag"
```

Comment:
why is this necessary? you can inherit this type from k8s or origin, or is it not possible?

Reply:
We don't have the `Kind` in string format, just the type in the `openshift/api`. To use as string we should rely on reflection:

https://github.com/openshift/api/blob/master/image/v1/register.go

Do you know any place where those Kinds are available as `string`? This can come in handy for a nice refactor to our API.

Reply:
There's something similar on 
 

> "github.com/openshift/origin/pkg/oc/cli/create".create.ImageStreamTagRecommendedName

But not sure if make sense to use it.